mkposters/mkposter.py

```diff
@@ -98,12 +104,22 @@ def svg_load_fn(match):
 
 def max_file_time(path: pathlib.Path):
     times = []
+
+    if path.match(".*"):  # This should ignore hidden directories such as .git/ not sure if this breaks css
+        return None
```

Comment:
This function used to return `int`; now it returns `None | int`. This means you're now doing a couple of code smells like implicit-cast-to-bool.

I think you can skip this complexity by reverting this function to how it was before, and then changing:
```diff
-        if subpath.is_file():
-            times.append(subpath.stat().st_mtime)
-        elif subpath.is_dir():
-            times.append(max_file_time(subpath))
+        if not subpath.match(".*"):
+            if subpath.is_file():
+                times.append(subpath.stat().st_mtime)
+            elif subpath.is_dir() and (len(subpath.iterdir()) > 0):
+                times.append(max_file_time(subpath))
```